command/title_body_survey.go

```diff
@@ -99,30 +107,36 @@ func confirmSubmission(allowPreview bool, allowMetadata bool) (Action, error) {
 	}
 }
 
-func selectTemplate(templatePaths []string) (string, error) {
+func selectTemplate(templatePaths []string, metadataType metadataStateType) (string, error) {
 	templateResponse := struct {
 		Index int
 	}{}
-	if len(templatePaths) > 1 {
-		templateNames := make([]string, 0, len(templatePaths))
-		for _, p := range templatePaths {
-			templateNames = append(templateNames, githubtemplate.ExtractName(p))
-		}
+	templateNames := make([]string, 0, len(templatePaths))
+	for _, p := range templatePaths {
+		templateNames = append(templateNames, githubtemplate.ExtractName(p))
+	}
+	if metadataType == issueMetadata {
+		templateNames = append(templateNames, "Open a blank issue")
+	} else if metadataType == prMetadata {
+		templateNames = append(templateNames, "Open a blank PR")
```

Comment:
We try to not use "PR" abbreviation in public-facing text, so this should be "pull request" instead of PR

Reply:
Okay. I will change that